pkg/controllers/exceptions/controller.go

```diff
@@ -155,7 +155,7 @@ func (c *controller) buildRuleIndex(key string, policy kyvernov1.PolicyInterface
 		return 0
 	})
 	index := ruleIndex{}
-	for _, name := range autogenv2.GetAutogenRuleNames(policy) {
+	for _, name := range autogen.Default.GetAutogenRuleNames(policy) {
```

Comment:
why not `autogen.V2` instead of `autogen.Default`?

Reply:
We don't want coupling with v2, next step will be to remove `Default`, `V1`, and `V2` and make it configurable (with a flag or toggle).